src/answers-umd.js

```diff
@@ -237,37 +237,37 @@ class Answers {
 
     this._onReady = parsedConfig.onReady || function () {};
 
-    const callPonyfillCssVariables = (callback) => {
-      if (!parsedConfig.disableCssVariablesPonyfill) {
-        this.ponyfillCssVariables({
-          onFinally: () => {
-            callback();
-          }
-        });
-      } else {
-        callback();
-      }
-    };
-
     if (parsedConfig.useTemplates === false || parsedConfig.templateBundle) {
       if (parsedConfig.templateBundle) {
         this.renderer.init(parsedConfig.templateBundle);
       }
 
-      callPonyfillCssVariables(this._onReady.bind(this));
+      this._handlePonyfillCssVariables(parsedConfig, this._onReady.bind(this));
       return this;
     }
 
     // Templates are currently downloaded separately from the CORE and UI bundle.
     // Future enhancement is to ship the components with templates in a separate bundle.
     this.templates = new DefaultTemplatesLoader(templates => {
       this.renderer.init(templates);
-      callPonyfillCssVariables(this._onReady.bind(this));
+      this._handlePonyfillCssVariables(parsedConfig, this._onReady.bind(this));
     });
 
     return this;
   }
 
+  _handlePonyfillCssVariables (parsedConfig, callback) {
```

Comment:
Let's add some JSDoc for this function and it's params. Minor nit, but let's just pass in a ponyfillDisabled booelan. We don't need the entire parsedConfig passed in.